src/main/java/me/zeroeightsix/kami/event/ForgeEventProcessor.kt

```diff
@@ -34,7 +34,7 @@ object ForgeEventProcessor {
         KamiEventBus.post(event)
 
         if (mc.world != null && mc.player != null) {
-            SafeTickEvent(event.phase).also {
+            TickEvent.ClientTickEvent(event.phase).also {
                 KamiEventBus.post(it)
             }
         }
```

Comment:
These lines are for posting the `SafeTickEvent`. It should be removed if you already replaced all the usages and have the event class removed.

Reply:
The `TickEvent.ClientTickEvent` is already posted at line 34. Null checks will be done by `safeListener`